Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices.py

```diff
@@ -133,9 +137,13 @@ def clusters_addon_update(client: AKSClient, args: Dict) -> str:
 
 
 def start_auth(client: AKSClient) -> CommandResults:
```

Comment:
since `start_auth` is also common code for all the integration, lets move it to the api module as well